src/elements/tickets-block.html

```diff
@@ -16,113 +17,170 @@
-                  <div class="sold-out" block$="[[ticket.soldOut]]">{$ ticketsBlock.soldOut $}</div>
-                  <div class="price" sold-out$="[[ticket.soldOut]]">
-                    [[ticket.price]]
-                    <span class="currency">[[ticket.currency]]</span>
-                  </div>
-                </div>
-              </div>
-            </li>
-          </template>
-        </ul>
+      @media (min-width: 640px) {
+        .tickets-placeholder {
+          grid-template-columns: repeat(auto-fill, 200px);
+        }
 
-        <div class="additional-info">*{$ ticketsBlock.ticketsDetails $}</div>
+        .ticket-item {
+          max-width: 200px;
+        }
 
-        <a href$="[[tickets.url]]"
+        .ticket-item[in-demand] {
+          transform: scale(1.15);
+        }
+      }
+    </style>
+
+    <div class="tickets-wrapper container">
+      <h1 class="container-title">{$ ticketsBlock.title $}</h1>
+      <content-loader
+        class="tickets-placeholder"
+        card-padding="24px"
+        card-height="216px"
+        border-radius="var(--border-radius)"
+        title-top-position="32px"
+        title-height="42px"
+        title-width="70%"
+        load-from="-70%"
+        load-to="130%"
+        animation-time="1s"
+        items-count="{$ contentLoaders.tickets.itemsCount $}"
+        hidden$="[[contentLoaderVisibility]]">
+      </content-loader>
+
+      <div class="tickets" layout horizontal wrap center-justified>
+        <template is="dom-repeat" items="[[tickets]]" as="ticket">
+          <a
+            class="ticket-item card"
+            href$="[[ticket.url]]"
             target="_blank"
             rel="noopener noreferrer"
+            in-demand$="[[ticket.inDemand]]"
+            on-tap="_onTicketTap"
             ga-on="click"
-            ga-event-category="ticket button"
-            ga-event-action="buy_click">
-          <paper-button class="buy-button" primary>{$ buyTicket $}</paper-button>
-        </a>
+            ga-event-category="ticket"
+            ga-event-action="buy_click"
+            ga-event-label$="[[ticket.name]]"
+            layout vertical justified
+          >
+            <div class="header">
+              <h3>[[ticket.name]]</h3>
+            </div>
+            <div class="content">
+              <div class="ticket-price-wrapper" layout vertical center-center>
+                <div class="price">[[ticket.currency]][[ticket.price]]</div>
+                <div class="discount">[[_getDiscount(ticket)]]</div>
+              </div>
+              <div class="type-description">
+                <div class="ticket-dates" hidden$="[[!ticket.starts]]">[[ticket.starts]] - [[ticket.ends]]</div>
+                <div class="ticket-info">[[ticket.info]]</div>
+              </div>
+            </div>
+            <div class="actions">
+              <div class="sold-out" block$="[[ticket.soldOut]]">
```

Comment:
what does the 'block' attribute mean?

Reply:
https://github.com/gdg-x/hoverboard/blob/hoverboard-v2/src/elements/flex-layout-attr.html#L353